content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
++----------+----------+------------+------------+
+|   HASH   | REVISION | TOTAL KEYS | TOTAL SIZE |
++----------+----------+------------+------------+
+| fe01cf57 |       10 |          7 | 2.1 MB     |
++----------+----------+------------+------------+
+```
+
+### Snapshot volume
+
+Jika etcd berjalan pada volume penyimpanan yang mendukung cadangan, seperti Amazon Elastic Block Store, buat cadangan data etcd dengan mengambil snapshot dari volume penyimpanan.
+
+## Scaling up klaster etcd
+
+Penskalaan klaster etcd meningkatkan ketersediaan dengan menukar kinerja. Penskalaan tidak meningkatkan kinerja atau kemampuan klaster. Aturan umum adalah untuk tidak _scale up_ atau _scale down_ klaster etcd. Jangan mengkonfigurasi grup penskalaan otomatis untuk klaster etcd. Sangat disarankan untuk selalu menjalankan klaster etcd five-member statis untuk klaster Kubernetes produksi pada skala yang didukung secara resmi.
+
+Penskalaan yang wajar adalah untuk meningkatkan klaster three-member menjadi five-member, ketika dibutuhkan lebih banyak reliabilitas. Lihat [Dokumentasi Rekonfigurasi etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/runtime-configuration.md#remove-a-member) untuk informasi tentang cara menambahkan anggota ke klaster yang ada.
+
+## Memulihkan klaster etcd
+
+etcd mendukung pemulihan dari snapshot yang diambil dari proses etcd dari versi [major.minor](http://semver.org/). Memulihkan versi dari versi patch lain dari etcd juga didukung. Operasi pemulihan digunakan untuk memulihkan data klaster yang gagal.
+
+Sebelum memulai operasi pemulihan, file snapshot harus ada. Ini bisa berupa file snapshot dari operasi cadangan sebelumnya, atau dari yang sisa [direktori data](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir). Untuk informasi dan contoh lebih lanjut tentang memulihkan klaster dari file snapshot, lihat [dokumentasi pemulihan bencana etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/recovery.md#restoring-a-cluster).
+
+Jika URL akses dari klaster yang dipulihkan diubah dari klaster sebelumnya, server API Kubernetes harus dikonfigurasi ulang dengan sesuai dengan itu. Pada kasus ini, restart server API Kubernetes dengan flag `--etcd-servers=$NEW_ETCD_CLUSTER` bukan flag `--etcd-servers=$OLD_ETCD_CLUSTER`. Ganti `$NEW_ETCD_CLUSTER` dan `$OLD_ETCD_CLUSTER` dengan alamat IP masing-masing. Jika load balancer digunakan di depan klaster etcd, kamu mungkin perlu memperbarui penyeimbang beban sebagai gantinya.
+
+Jika mayoritas anggota etcd telah gagal secara permanen, klaster etcd dianggap gagal. Dalam skenario ini, Kubernetes tidak dapat membuat perubahan apa pun ke kondisi saat ini. Meskipun _Pod_ terjadwal mungkin terus berjalan, tidak ada _Pod_ baru yang bisa dijadwalkan. Dalam kasus seperti itu, pulihkan klaster etcd dan kemungkinan juga untuk mengkonfigurasi ulang server API Kubernetes untuk memperbaiki masalah ini.
```

Comment:
"Pod" is an API name, it is written only with the capital letter, no need italics.
```suggestion
Jika mayoritas anggota etcd telah gagal secara permanen, klaster etcd dianggap gagal. Dalam skenario ini, Kubernetes tidak dapat membuat perubahan apa pun ke kondisi saat ini. Meskipun Pod terjadwal mungkin terus berjalan, tidak ada Pod baru yang bisa dijadwalkan. Dalam kasus seperti itu, pulihkan klaster etcd dan kemungkinan juga untuk mengonfigurasi ulang server API Kubernetes untuk memperbaiki masalah ini.
```